test/passing/break_cases_all.ml.ref

```diff
@@ -162,3 +162,22 @@ let ffffff ~foo =
   | Eeeeeeeeeeeeeeeee ->
       foooooooooooooooooooo
   | Fffffffffffffffff -> fooooooooooooooooo
+
+let _ = function
+  | (exception A)
```

Comment:
Do we need the parens?

Reply:
It's handled in the same way as `lazy` (which is also parenthesized), it's not a regression of this PR and it can be fixed in another one.